src/components/container/DailyDataChart/DailyDataChart.tsx

```diff
@@ -30,12 +31,19 @@ export interface LineChartOptions {
 }
 
 export interface BarChartOptions {
-    barColor?: string
+    barColor?: ColorDefinition
+    thresholds?: BarChartThreshold[]
+}
+
+export interface BarChartThreshold {
```

Comment:
@chrisnowak Something coming to mind, could there be an option where there is a need to highlight under, within, and over range

Reply:
I'm mostly thinking about under range. The other 2 (within and over) are already taken care of.

Reply:
The default color would apply when under any of the thresholds - so you can totally achieve that with this pretty easily.  E.g. if you want someone to walk between 5000 and 10000 steps per day and highlight days they did it in green, you could set default color to grey, a threshold for 5,000 with a color of green, and then a threshold for 10k with grey again